src/components/Toast.js

```diff
@@ -11,7 +11,10 @@ export default function Toast({ type = "info", message }) {
           type === "error" && "alert-error",
         ])}
       >
-        <span>{message}</span>
+        <span className="font-bold">
+          {message.slice(0, 18)}
+          <span className="text-white ">{message.slice(18)}</span>
+        </span>
```

Comment:
This will not work for any other message - further improvements to this can come later, let's get it merged sooner before adding more changes

Reply:
Ok

Reply:
Updated Toast.js